examples/logon_logoff.py

```diff
@@ -5,7 +5,7 @@
     configure_single_client_tes_connection
 from tes_client.messaging.common_types import AccountCredentials, AccountInfo
 from tes_client.messaging.printing_response_handler import \
-    PrintingResponseHandler
+    PrintingResponseHandler, SessionRefresher
```

Comment:
[Suggestion]
I think SessionRefresher needs to be moved out of scope of this lib. It will not work for multiple clients and cannot server as an example since clients can implement it different way depends on their platform and codebase.

Reply:
Renamed to SingleClientSessionRefresher and moved it into examples. I made it clear in the comment documentation that this implementation is only useful for a single user client. Multi-user clients need to develop their own solution. I think it is helpful as an example for single user clients, like some funds.